depends/README.md

```diff
@@ -41,7 +41,7 @@ The paths are automatically configured and no other options are needed.
 
 #### Common
 
-    apt install bison cmake curl make patch pkg-config python3 xz-utils
+    apt install bison cmake curl make ninja-build patch pkg-config python3 xz-utils
```

Comment:
There's probably enough GUI-only stuff here, i.e `bison, ninja-build, python3, xz-utils`, that this could be moved to it's own `#### Gui` section.

Reply:
See https://github.com/bitcoin/bitcoin/pull/31192.